<launch>
  <node pkg="pointgrey_camera_driver" type="camera_node" name="barc_cam" output="screen">
  <param name="framerate" value="30"/>    
  <param name="image_transport" value="compressed" />
  </node>

  <node pkg="goldeneye" type="redbox.py" name="redbox_bounds" output="screen">
  </node>

  <arg name="display_image" default="False" />
  <arg name="display_processed_image" default="False" />
  <arg name="publish_processed_image" default="False" />
  <arg name="debug_info" default="False" />

  <param name="display_image" value="$(arg display_image)" type="boolean"/>
  <param name="display_processed_image" value="$(arg display_processed_image)" type="boolean"/>
  <param name="publish_processed_image" value="$(arg publish_processed_image)" type="boolean"/>
  <param name="debug_info" value="$(arg debug_info)" type="boolean"/>

</launch>
